build/build.sh

```diff
@@ -135,8 +129,6 @@ function wait_for_mysql_postgres() {
 function wait_for_bigdata() {
     show_mem
 
-    sbt scalariformFormat test:scalariformFormat
```

Comment:
I don't know what this is for but I doubt its neccessary

Reply:
This formats the tests. Doesn’t need to be there though